features/serviceworker-blocking.json

```diff
@@ -0,0 +1,11 @@
+{
+    "_meta": {
+        "description": "Service worker blocking. Blocking of requests initiated by service workers",
```

Comment:
Maybe ditch the first sentence and expand the rest a bit? E.g. "Blocking of tracking requests initiated by service workers. Considered a separate feature since it has caused breakage in the past."